src/vs/base/browser/ui/iconLabel/iconlabel.css

```diff
@@ -31,6 +31,9 @@
 	flex-shrink: 0; /* fix for https://github.com/microsoft/vscode/issues/13787 */
 }
 
+.monaco-icon-label-container.disabled {
+	opacity: 0.60;
```

Comment:
Does using `var(--vscode-disabledForeground)` work here? Similar to 

https://github.com/microsoft/vscode/blob/34f184263de048a6283af1d9eb9faab84da4547d/src/vs/editor/contrib/find/browser/findWidget.css#L169-L170

It seems like .6 is fairly standard throughout the codebase, but if the `var` will work then it should play better with themes.

Reply:
Yes, that would be better. How about the Icons? Probably better to only adapt the opacity instead of the color as it's not possible to change the color of all icons. Just changing where possible would create an inconsistency